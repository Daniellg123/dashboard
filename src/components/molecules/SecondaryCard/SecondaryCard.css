.card1 {
    box-sizing: border-box;

    /* Auto layout */
    display: flex;
    flex-direction: column;
    padding: 12px;
    gap: 10px;

    position: static;
    width: 387px;
    height: 140px;

    /* Background */
    background: #FAFCFF;

    /* Orange/50 Primary */
    border: 1px solid #ED5E2E;
}

.card2 {
    box-sizing: border-box;

    /* Auto layout */
    display: flex;
    flex-direction: column;
    padding: 12px;
    gap: 10px;

    position: absolute;
    width: 195px;
    height: 140px;

    /* Background */
    background: #FAFCFF;

    /* Gray/50 */
    border: 1px solid #ABB0B9;
}

.content {
    display: flex;
    flex-direction: row;
    gap: 10px;
    align-items: center;
}

.c {
    display: block;
    width: 50%;
    display: flex;
    flex-direction: row;
    gap: 10px;
    align-items: center;
}

.subInfoCard {
    display: flex;
    flex-direction: column;
}