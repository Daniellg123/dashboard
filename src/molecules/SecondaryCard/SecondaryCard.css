.card {
    box-sizing: border-box;

    /* Auto layout */
    display: flex;
    flex-direction: column;
    padding: 12px;
    gap: 10px;

    position: absolute;
    width: 387px;
    height: 140px;

    /* Background */
    background: #FAFCFF;

    /* Orange/50 Primary */
    border: 1px solid #ED5E2E;
}

.content {
    display: flex;
    flex-direction: row;
    gap: 10px;
}

.infoP {
    display: flex;
    flex-direction: row;
    gap: 10px;
}