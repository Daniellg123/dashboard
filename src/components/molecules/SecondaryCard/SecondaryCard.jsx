import PropTypes from 'prop-types';
import { Card, Typography, Chip, Divider, Avatar, AvatarGroup } from '@mui/material';
import ImageCard from '../../atoms/ImageCard/ImageCard';
import InfoCard from '../InfoCard/InfoCard';
import PrincipalCard from '../PrincipalCard/PrincipalCard';

import './SecondaryCard.css';

/**
 * Componente para mostrar más información
 */

function SecondaryCard({ img, imgP, title, users, units, name, position, selected }) {
    return (
        <>
            {!selected ?
                (<PrincipalCard img={img} title={title} users={users} units={units} />)
                :
                (
                    <div className='subInfoCard'>
                        <Card className='card1'>
                            <div className='content'>
                                <div className='c'>
                                    <ImageCard imgDir={img} size={48} />
                                    <Typography variant='body2'>{title}</Typography>
                                </div>
                                <div className='c'>
                                    <Chip label={`${users + ' Total Users'}`} />
                                </div>
                            </div>
                            <Divider variant='middle' />
                            <div className='content'>
                                <div className='c'>
                                    <ImageCard imgDir={imgP} size={24} />
                                    <div>
                                        <Typography variant='body1'>{name}</Typography>
                                        <Typography variant='body3'>{position}</Typography>
                                    </div>
                                </div>
                                <div className='c'>
                                    <Typography variant='body1'>Users ({users})</Typography>
                                    <AvatarGroup >
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                        <Avatar src='' sx={{ height: '24px', width: '24px' }} />
                                    </AvatarGroup>
                                </div>
                            </div>
                        </Card>
                        <InfoCard units={units} users={users} />
                    </div>
                )
            }
        </>
    );
};

SecondaryCard.propTypes = {
    /**
     * Ruta de la imagen del Card
     */
    img: PropTypes.string, 
    /**
     * Ruta de la imagen de la persona
     */
    imgP: PropTypes.string,
    /**
     * Titulo del card
     */
    title: PropTypes.string, 
    /**
     * Numero de users
     */
    users: PropTypes.number, 
    /**
     * Númerp de units
     */
    units: PropTypes.number, 
    /**
     * Nombre de la persona
     */
    name: PropTypes.string, 
    /**
     * Cargo de la persona
     */
    position: PropTypes.string, 
    /**
     * Booleano para saber si fue seleccionado
     */
    selected: PropTypes.bool
};

export default SecondaryCard;