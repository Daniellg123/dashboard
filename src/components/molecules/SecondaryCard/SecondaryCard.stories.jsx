import SecondaryCard from "./SecondaryCard";

const routeImg = './img/Imageofemployee.png';
const routeImg2 = './img/logo1.png';

export default {
    title: 'Molecules/SecondaryCard',
    component: SecondaryCard,
};

const Template = (args) => <SecondaryCard {...args} />

export const Primary = Template.bind({});
Primary.args = {img: routeImg, imgP: routeImg2 , title: 'Dise√±o UX', users: 300, units: 43, name: 'Daniel', position: 'Maestro', selected: false};