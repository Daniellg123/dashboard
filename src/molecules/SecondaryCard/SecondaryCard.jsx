import { Card, Typography, Chip, Divider, Avatar, AvatarGroup } from '@mui/material';
import ImageCard from '../../atoms/ImageCard/ImageCard';

import PrincipalCard from '../PrincipalCard/PrincipalCard';

import './SecondaryCard.css';

function SecondaryCard({ img, imgP, title, users, units, name, position, selected }) {
    return (
        <>
            {selected ? 
                (<PrincipalCard img={img} title={title} users={users} units={units} />)
            :
                (
                    <Card className='card1'>
                        <div className='content'>
                            <div className='c'>
                                <ImageCard imgDir={img} size={48}/>
                                <Typography variant='body2'>{title}</Typography>
                            </div>
                            <div className='c'>
                                <Chip label={`${users + ' Total Users'}`} />
                            </div>
                        </div>
                        <Divider variant='middle'/>
                        <div className='content'>
                            <div className='c'>
                                <ImageCard imgDir={imgP} size={24}/>
                                <div>
                                    <Typography variant='body1'>{name}</Typography>
                                    <Typography variant='body3'>{position}</Typography>
                                </div>
                            </div>
                            <div className='c'>
                                <Typography variant='body1'>Users ({users})</Typography>
                                <AvatarGroup >
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                                    <Avatar src='' sx={{ height: '24px', width: '24px' }}/>                        
                                </AvatarGroup>
                            </div>
                        </div>
                    </Card>
                )
            }
        </>
    );
};

export default SecondaryCard;


/*
<PrincipalCard img={img} title={title} users={users} units={units} />


<Card className='card1'>
            <div className='content'>
                <div className='c'>
                    <ImageCard imgDir={img} size={48}/>
                    <Typography variant='body2'>{title}</Typography>
                </div>
                <div className='c'>
                    <Chip label={`${users + ' Total Users'}`} />
                </div>
            </div>
            <Divider variant='middle'/>
            <div className='content'>
                <div className='c'>
                    <ImageCard imgDir={img} size={24}/>
                    <div>
                        <Typography variant='body1'>{name}</Typography>
                        <Typography variant='body3'>{position}</Typography>
                    </div>
                </div>
                <div className='c'>
                    <Typography variant='body1'>Users ({users})</Typography>
                    <AvatarGroup >
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>
                        <Avatar src='' sx={{ height: '24px', width: '24px' }}/>                        
                    </AvatarGroup>
                </div>
            </div>
        </Card>
*/