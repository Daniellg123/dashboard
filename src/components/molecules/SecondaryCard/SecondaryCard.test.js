import { render } from '@testing-library/react';
import SecondaryCard from './SecondaryCard';

const props = {
    img: './img/logo1.png', 
    imgP: './img/Avatar.png', 
    title: 'Titulo',
    name: 'Daniel', 
    position: 'Profesor', 
    selected: true,
    units: 12,
    users: 300,
};

describe(' Secondary card selected', () => {
    it('renders correctly', () => {
        const component = render(
            <SecondaryCard {...props}/>
        );
        expect(component).toMatchSnapshot();
    });
    
    it('Units is equal to prop users', () => {
        const component = render(<SecondaryCard {...props} users={14}/>)
        component.getByText('14 Total Users');
    });

    it('Name is equal to prop name', () => {
        const component = render(<SecondaryCard {...props} name={'Oswaldo'}/>)
        component.getByText('Oswaldo');
    });
});

describe(' Secondary card no selected', () => {
    it('renders correctly', () => {
        const component = render(
            <SecondaryCard {...props} selected={false}/>
        );
        expect(component).toMatchSnapshot();
    });
    
    it('Units is equal to prop units', () => {
        const component = render(<SecondaryCard {...props} selected={false} units={14}/>)
        component.getByText('14');
    });
    
    it('Units is equal to prop users', () => {
        const component = render(<SecondaryCard {...props} selected={false} users={14}/>)
        component.getByText('14');
    });
});

