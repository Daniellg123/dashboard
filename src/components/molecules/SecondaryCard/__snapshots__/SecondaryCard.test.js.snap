// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Secondary card no selected renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-1hs4s8w-MuiPaper-root-MuiCard-root"
      >
        <div
          class="content"
        >
          <img
            alt="./img/logo1.png"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-14w8qym-MuiCardMedia-root"
            src="./img/logo1.png"
          />
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            Titulo
          </p>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
        />
        <div
          class="content"
        >
          <div
            class="info"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            >
              12
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Units
            </p>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-fullWidth MuiDivider-vertical css-ftc7wc-MuiDivider-root"
          />
          <div
            class="info"
          >
            <p
              class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            >
              300
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Users
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-1hs4s8w-MuiPaper-root-MuiCard-root"
    >
      <div
        class="content"
      >
        <img
          alt="./img/logo1.png"
          class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-14w8qym-MuiCardMedia-root"
          src="./img/logo1.png"
        />
        <p
          class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
        >
          Titulo
        </p>
      </div>
      <hr
        class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
      />
      <div
        class="content"
      >
        <div
          class="info"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            12
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Units
          </p>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth MuiDivider-vertical css-ftc7wc-MuiDivider-root"
        />
        <div
          class="info"
        >
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
          >
            300
          </p>
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Users
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[` Secondary card selected renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="subInfoCard"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card1 css-bhp9pd-MuiPaper-root-MuiCard-root"
        >
          <div
            class="content"
          >
            <div
              class="c"
            >
              <img
                alt="./img/logo1.png"
                class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-14w8qym-MuiCardMedia-root"
                src="./img/logo1.png"
              />
              <p
                class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Titulo
              </p>
            </div>
            <div
              class="c"
            >
              <div
                class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-gac2fo-MuiChip-root"
              >
                <span
                  class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
                >
                  300 Total Users
                </span>
              </div>
            </div>
          </div>
          <hr
            class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
          />
          <div
            class="content"
          >
            <div
              class="c"
            >
              <img
                alt="./img/Avatar.png"
                class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-1tb3t5s-MuiCardMedia-root"
                src="./img/Avatar.png"
              />
              <div>
                <p
                  class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                >
                  Daniel
                </p>
                <span
                  class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
                >
                  Profesor
                </span>
              </div>
            </div>
            <div
              class="c"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Users (
                300
                )
              </p>
              <div
                class="MuiAvatarGroup-root css-1ytufz-MuiAvatarGroup-root"
              >
                <div
                  class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-sxh3gq-MuiAvatar-root-MuiAvatarGroup-avatar"
                >
                  +
                  7
                </div>
                <div
                  class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                    data-testid="PersonIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                    data-testid="PersonIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                    data-testid="PersonIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                    />
                  </svg>
                </div>
                <div
                  class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                    data-testid="PersonIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root infoCard css-bhp9pd-MuiPaper-root-MuiCard-root"
        >
          <span
            class="MuiTypography-root MuiTypography-body4 css-37jdci-MuiTypography-root"
          >
            Child Hierarchy
          </span>
          <div
            class="infoSub"
          >
            <span
              class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
            >
              12
               Units
            </span>
            <hr
              class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
            />
            <span
              class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
            >
              300
               Users
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="subInfoCard"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card1 css-bhp9pd-MuiPaper-root-MuiCard-root"
      >
        <div
          class="content"
        >
          <div
            class="c"
          >
            <img
              alt="./img/logo1.png"
              class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-14w8qym-MuiCardMedia-root"
              src="./img/logo1.png"
            />
            <p
              class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            >
              Titulo
            </p>
          </div>
          <div
            class="c"
          >
            <div
              class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-gac2fo-MuiChip-root"
            >
              <span
                class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
              >
                300 Total Users
              </span>
            </div>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
        />
        <div
          class="content"
        >
          <div
            class="c"
          >
            <img
              alt="./img/Avatar.png"
              class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img img css-1tb3t5s-MuiCardMedia-root"
              src="./img/Avatar.png"
            />
            <div>
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Daniel
              </p>
              <span
                class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
              >
                Profesor
              </span>
            </div>
          </div>
          <div
            class="c"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Users (
              300
              )
            </p>
            <div
              class="MuiAvatarGroup-root css-1ytufz-MuiAvatarGroup-root"
            >
              <div
                class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-sxh3gq-MuiAvatar-root-MuiAvatarGroup-avatar"
              >
                +
                7
              </div>
              <div
                class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                  data-testid="PersonIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                  />
                </svg>
              </div>
              <div
                class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                  data-testid="PersonIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                  />
                </svg>
              </div>
              <div
                class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                  data-testid="PersonIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                  />
                </svg>
              </div>
              <div
                class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault MuiAvatarGroup-avatar css-irctgw-MuiAvatar-root"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiAvatar-fallback css-10mi8st-MuiSvgIcon-root-MuiAvatar-fallback"
                  data-testid="PersonIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root infoCard css-bhp9pd-MuiPaper-root-MuiCard-root"
      >
        <span
          class="MuiTypography-root MuiTypography-body4 css-37jdci-MuiTypography-root"
        >
          Child Hierarchy
        </span>
        <div
          class="infoSub"
        >
          <span
            class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
          >
            12
             Units
          </span>
          <hr
            class="MuiDivider-root MuiDivider-middle css-ft4jij-MuiDivider-root"
          />
          <span
            class="MuiTypography-root MuiTypography-body3 css-37jdci-MuiTypography-root"
          >
            300
             Users
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
